schemas/ispyb/stored_programs/sp_retrieve_image_quality_indicators.sql

```diff
@@ -0,0 +1,24 @@
+DELIMITER ;;
+CREATE OR REPLACE DEFINER=`ispyb_root`@`%` PROCEDURE `retrieve_image_quality_indicators`(p_dataCollectionId int unsigned,
```

Comment:
I suggest renaming it to `retrieve_quality_indicators` (drop the `image_`) to harmonise with the existing `upsert_quality_indicators` procedure.